presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcConnection.java

```diff
@@ -186,6 +187,28 @@ public void testSession()
             assertThat(listSession(connection))
                     .contains("join_distribution_type|BROADCAST|PARTITIONED")
                     .contains("exchange_compression|true|false");
+
+            try (Statement statement = connection.createStatement()) {
```

Comment:
please extract this as separate test

Reply:
Why?

Reply:
This is just my feeling. You are added tests that are very focused on characters that are sent in headers for Presto session, while the test name is very general `testSession` which suggests (to me) it is a smoke kind of test.

Reply:
The test is focused on `SET SESSION` and `SHOW SESSION` statement. I will leave it where it is